sdk/formrecognizer/Azure.AI.FormRecognizer/api/Azure.AI.FormRecognizer.netstandard2.0.cs

```diff
@@ -501,19 +499,20 @@ public partial class DocumentFieldSchema
     }
     public enum DocumentFieldType
     {
-        String = 0,
-        Date = 1,
-        Time = 2,
-        PhoneNumber = 3,
-        Double = 4,
-        Int64 = 5,
-        List = 6,
-        Dictionary = 7,
-        SelectionMark = 8,
-        CountryRegion = 9,
-        Signature = 10,
-        Currency = 11,
-        Address = 12,
+        Unknown = 0,
```

Comment:
Isn't this a breaking change? Should we not mention it explicitly in changelog?

Reply:
It's mentioned in the CHANGELOG:
> `DocumentField.ValueType` (now called `FieldType`) can now be `Unknown` when the field value couldn't be parsed by the service.